src/components/Station.vue

```diff
@@ -97,4 +111,16 @@ const { isFetching, error, data } = useFetch(
   ),
   { refetch: true }
 ).json<StationGeoJSONSerializer>();
+
+function filterSelectedParams(parameter: ParameterMetadataModel) {
+  return parameters.value.indexOf(parameter.name) > -1;
+}
 </script>
+<style>
+.no-border-radius {
+  border-radius: unset;
+}
```

Comment:
There's `rounded-0` in Bootstrap: https://getbootstrap.com/docs/5.2/utilities/borders/#sizes